component/src/main/java/com/synopsys/integration/alert/component/diagnostic/database/DefaultDiagnosticAccessor.java

```diff
@@ -58,4 +69,13 @@ private AuditDiagnosticModel getAuditDiagnosticInfo() {
             auditEntryRepository.getAverageAuditEntryCompletionTime().orElse(AuditDiagnosticModel.NO_AUDIT_CONTENT_MESSAGE)
         );
     }
+
+    private SystemDiagnosticModel getSystemInfo() {
+        Runtime runtime = Runtime.getRuntime();
+        return new SystemDiagnosticModel(runtime.availableProcessors(), runtime.maxMemory(), runtime.totalMemory(), runtime.freeMemory());
+    }
+
+    private RabbitMQDiagnosticModel getRabbitMQDiagnosticInfo() {
+        return rabbitMQDiagnosticUtility.getRabbitMQDiagnostics();
+    }
```

Comment:
```suggestion
```
Since we just use the utility call it above and remove this get method. If how we get the metrics changes then we can add this method.